CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 
 #### 4.x Releases
 
-- `4.1.x` Releases - [4.1.0](#410)
+- `4.1.x` Releases - [4.1.0](#410)|| [4.1.1](#411)
```

Comment:
Extra `|`. Should be `(#410) | [4.1.1](#411)`.

Reply:
Done